virtualcluster/cmd/vn-agent/app/server.go

```diff
@@ -103,6 +104,8 @@ func Run(c *config.Config, serverOption *options.ServerOption, stopCh <-chan str
 		},
 	}
 
+	fmt.Printf(serverOption.ClientCAFile)
```

Comment:
Why we wanna print out the client CA file?

Reply:
Doh, debugging lol, I will remove that.